scrapy/extensions/feedexport.py

```diff
@@ -151,7 +151,7 @@ def __init__(self, settings):
         if not self._exporter_supported(self.format):
             raise NotConfigured
         self.store_empty = settings.getbool('FEED_STORE_EMPTY')
-        self.export_fields = settings.getlist('FEED_EXPORT_FIELDS')
+        self.export_fields = settings.getlist('FEED_EXPORT_FIELDS') or None
```

Comment:
It was not possible to disable feed export fields by setting something to `[]` before introducing `FEED_EXPORT_FIELDS` option, and it is not possible now, so I don't see a problem :) But ok, I'll fix it.